core/flyout_base.ts

```diff
@@ -615,7 +615,7 @@ export abstract class Flyout extends DeleteArea implements IFlyout {
     }
 
     this.listeners_.push(browserEvents.conditionalBind(
-        (this.svgBackground_ as SVGPathElement), 'mouseover', this,
+        (this.svgBackground_ as SVGPathElement), 'pointerover', this,
```

Comment:
(no action needed): One day we'll just delete this, because it's a workaround for an IE issue that we no longer care about.